src/Microsoft.PowerShell.Commands.Management/commands/management/PingPathCommand.cs

```diff
@@ -109,7 +109,7 @@ public override string[] Exclude
         /// </summary>
         [Parameter]
         [Alias("Type")]
-        public TestPathType PathType { get; set; } = TestPathType.Any;
```

Comment:
The change looks unreliable. If we change the enum we can break something.

Please revert (and local suppress warning) al  such changes.

Reply:
`Microsoft.PowerShell.Commands.TestPathType` is public API. Changing the zero value of the enum would be a breaking change so I don't think we need to revert.

Reply:
It would be a breaking change for binary only and it's too unlikely. 
More importantly, the code depends on this value. And for a better understanding of the code, it's better to have an explicit value than guess what design is implied.

Reply:
I say about all such cases.